packages/mdc-tab-scroller/util.ts

```diff
@@ -54,14 +50,20 @@ function computeHorizontalScrollbarHeight(documentObj, shouldCacheResult = true)
   return horizontalScrollbarHeight;
 }
 
-/**
- * @param {!Object} HTMLElementPrototype
- * @return {string}
- */
-function getMatchesProperty(HTMLElementPrototype) {
-  return [
-    'msMatchesSelector', 'matches',
-  ].filter((p) => p in HTMLElementPrototype).pop();
-}
+export type VendorMatchesFunctionName = 'webkitMatchesSelector' | 'msMatchesSelector';
+export type MatchesFunctionName = VendorMatchesFunctionName | 'matches';
 
-export {computeHorizontalScrollbarHeight, getMatchesProperty};
+export function getMatchesProperty(htmlElementPrototype: {}): MatchesFunctionName {
```

Comment:
shouldn't we be using the `@material/dom/matches` function here instead?

Reply:
probably can be a different PR actuallly... nvm

Reply:
I actually already updated `index.ts` to use `ponyfill.matches()`, but I'm leaving `util.getMatchesProperty()` here to avoid breaking backward compatibility for any clients who might be using it. We'll remove `getMatchesProperty()` from the various packages that provide it (including ripple) after all the TS stuff is merged, with a BREAKING CHANGE commit.